### PR TITLE
MF-253: Get translations working

### PR DESCRIPTION
**Pull request by brandones**
The problem was with an old version of react-root-router.

https://issues.openmrs.org/browse/MF-253
https://issues.openmrs.org/browse/MF-189

### CODE REVIEW

File: src/root.component.tsx [hunk 1]
```diff
@@ -15,6 +15,6 @@ function Root() {
 }
 
 export default openmrsRootDecorator({
-  featureName: 'patient-registration',
+  featureName: 'Patient Registration',
```
Comment (brandones) [inline src/root.component.tsx:18]: Unrelated and unimportant

File: package.json [hunk 2]
```diff
@@ -38,7 +38,7 @@
     "@babel/preset-react": "^7.0.0",
     "@babel/preset-typescript": "^7.3.3",
     "@openmrs/esm-error-handling": "^1.1.0",
-    "@openmrs/react-root-decorator": "^2.0.0",
+    "@openmrs/react-root-decorator": "^3.2.0",
```
Comment (brandones) [inline package.json:41]: This is the key

**Comment by denniskigen** [comment]
Great find @brandones!

**Comment by nickjhill14** [comment]
Nice spot to fix the translations @brandones! I'm just going through cleaning up some branches, is it okay to delete this one?

**Comment by brandones** [comment]
Yep, done!

**Pull request merged**